packages/lending/src/protocols/aave-v2/tokens/data/mainnet.json

```diff
@@ -1,74 +1,32 @@
 {
-  "USDT": {
-    "chainId": 1,
-    "address": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
-    "decimals": 6,
-    "symbol": "USDT",
-    "name": "Tether USD"
-  },
   "aUSDT": {
     "chainId": 1,
     "address": "0x3Ed3B47Dd13EC9a98b44e6204A523E766B225811",
     "decimals": 6,
     "symbol": "aUSDT",
     "name": "Aave interest bearing USDT"
   },
```

Comment:
Is it possible we remove all the unused testing tokens for in all `/data/*.json`? Like what we did in `protocolink-logics` repo.

Reply:
Or is it possible to use `protocolink-logics`' `/data/*.json` so that we can keep no `/data/*.json` for each protocol in lending-sdk?

Reply:
This seems feasible, only radiant's gmETH and gmBTC would need to be set, the rest wouldn't need tokens.json.

Reply:
since in `protocolink-logics`, we already do something like this
```
// protocolink-logics
export const arbitrumTokens = {
  ...common.toTokenMap<ArbitrumTokenSymbols>(arbitrumTokensJSON),
  ...common.arbitrumTokens,
};
```

I think we can remove `*.json` and keep `src/protocols/radiant-v2/tokens/index.ts`, and change to
```
// lending sdk
export const arbitrumTokens = logics.radiantv2.arbitrumTokens;

// if we still need other token
export const arbitrumTokens = {...logics.radiantv2.arbitrumTokens, ...otherTokens};
```

Reply:
https://github.com/dinngo/protocolink-js-sdk/pull/63/commits/34c81a23beeb102d756da8e44cfa5f3be376b85f

If `protocolink-logics` has defined tokens, then use `logics.xxxxxx.yyyyyyTokens`, otherwise, use the `common.yyyyyTokens`